.CONTRIBUTING.md

```diff
@@ -0,0 +1,316 @@
+# Contribute to the Polkadot Docs
+
+> **📋 Essential**: All contributions must follow the [PaperMoon Style Guide](https://github.com/papermoonio/documentation-style-guide).
+
+## Contents
+
+- [Quick Start](#quick-start)
+- [Edit Online with GitHub (Simple Changes)](#edit-online-with-github-simple-changes)
+- [Fork and Edit Locally](#fork-and-edit-locally)
+   - [Fix or Improve Existing Content](#fix-or-improve-existing-content)
+   - [Add a New Page](#add-a-new-page)
+   - [Create a New Section](#create-a-new-section)
+   - [Write a Tutorial](#write-a-tutorial)
+   - [Working with Content Elements](#working-with-content-elements)
+      - [Adding Images](#adding-images)
+      - [Using Code Snippets](#using-code-snippets)
+      - [Adding Callout Boxes](#adding-callout-boxes)
+
+## Quick Start
+
+There are two main ways to contribute:
+
+- **[Edit Online with GitHub](#edit-online-with-github-simple-changes)** 
+   - Best for simple text changes like fixing typos or making small updates directly in your browser.  
+
+- **[Fork and Edit Locally](#fork-and-edit-locally)** 
+   - Recommended for more complex contributions that require local testing or previewing changes.
+
+
+## Edit Online with GitHub (Simple Changes)
+
+For simple contributions like fixing typos or making small text changes, you can use GitHub's online editor.
+
+### Making Your Changes
+
+1. Navigate to the file you want to edit on GitHub.
+2. Click the pencil icon (✏️) to edit the file.
+3. Make your changes in the online editor.
+4. Follow the [PaperMoon Style Guide](https://github.com/papermoonio/documentation-style-guide).
+
+### Creating Your Pull Request
+
+1. Scroll to **Propose changes** at the bottom of the editor.
+2. Add a descriptive commit message explaining your changes.
+3. Click **Propose changes**. GitHub will automatically:
+   - Fork the repository to your account.
+   - Create a branch with your changes.
+   - Open a pull request.
+      - Make sure to check "Allow edits from maintainers"
+
+## Fork and Edit Locally
+
+For more complex contributions, which involve rendering (e.g., adding a `.Nav` section, code snippets, etc) or when you want to preview changes locally.
```

Comment:
```suggestion
For more complex contributions, which involve rendering (e.g., adding a `.nav.yml` section, code snippets, etc) or when you want to preview changes locally.
```

I'm not sure if we're trying to say a `.nav.yml` file or just generally speaking adding a section to the navigation, so this suggestion might not be correct.